src/clj/witan/send/send.clj

```diff
@@ -335,6 +330,14 @@
 
         valid-year-settings (->> (ds/row-maps valid-setting-academic-years)
                                  (states/calculate-valid-year-settings-from-setting-academic-years))]
+
+    (when (boolean (resolve 'u/log))
+      (when (not= 1 modify-transition-by)
+        (u/log-info (str "Modified transitions by " modify-transition-by)))
+      (if (nil? modified-transition-matrix)
```

Comment:
This is a new thing to me, but just `(if modified-transition-matrix then else)` should work here, just swap round the outputs

Reply:
@mikeyford Why do you need `(when (boolean (resolve 'u/log)) ...` ?

Reply:
@acron0 This was because the tests on the `defworkflow` this is in were failing because u/log didn't exist when it was tested independently.

Reply:
@mikeyford This suggests an issue with your dependencies. You shouldn't have to call `resolve` in code like this.